ros2topic/test/fixtures/repeater_node.py

```diff
@@ -37,14 +36,10 @@ def callback(self):
         self.pub.publish(self.message_type())
 
 
-def message_type(message_typename):
-    return import_message_type('~/output', message_typename)
-
-
 def parse_arguments(args=None):
     parser = argparse.ArgumentParser()
     parser.add_argument(
-        'message_type', type=message_type,
+        'message_type',
```

Comment:
@jacobperron fyi: the initial intent here was to have the argument parser fail right away and react accordingly if a given `message_type` is invalid or the actual message type instance cannot be looked up.

Reply:
If I change this line to have `type=get_message`, I run into an error as it seems the message object is passed (instead of a string).

Examples error: `AttributeError: type object 'BoundedSequences' has no attribute 'split'`

I'm not sure what's going on.

Reply:
Ah, I see now. I'm accidentally calling `get_message` twice. Fixed in 83bce83